stumpy/snippets.py

```diff
@@ -107,8 +107,8 @@ def _get_all_profiles(
             S_i,
             T,
             s,
-            M_T[start : start + s],
-            Σ_T[start : start + s],
+            M_T[start : start + m],
```

Comment:
Why should this be `m` instead of `s`?

Reply:
`M_T[start : start + ....]` should be the sliding mean of subseqs (of length `s`) in `S_i`. `S_i` itself has length `m`. So, I think a more accurate value for the sliding mean is `M_T[start : start + m - s + 1]` (I just realized this latter part). But since we iterate over subseqs of length `s` in `S_i`, `M_T[start : start + m]` works. 

However, it might be better to do `M_T[start : start + m - s + 1]` as I think it is clearer and more readable in this case.

Reply:
I see. Thank you for explaining it

Reply:
> However, it might be better to do `M_T[start : start + m - s + 1]` as I think it is clearer and more readable in this case.

If that is okay with you, I am going to use `M_T[start : start + m - s + 1]` as I think it forces reader to think (and hopefully remember the reason behind doing `+ m -  s  + 1`)

Reply:
Let's do whatever is the best option

Reply:
Perfect